stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 3.0.2
+version: 3.0.3
```

Comment:
We're adding functionality, so let's make this a minor (not patch) version bump.